### PR TITLE
[NOREF] QA homepage fixes

### PR DESCRIPTION
**Pull request by patrickseguraoddball**
# NOREF

## Description

Addressed items in the [QA Doc](https://docs.google.com/document/d/14Hs5FPIb4VCop0sdCLFT5PZn8VEy5G4Usg5Xic1VEJM/edit#heading=h.afwi1hqbmczu)

## How to test this change

- Verify all items in doc are addressed

## PR Author Checklist

- [ ] I have provided a detailed description of the changes in this PR.
- [ ] I have provided clear instructions on how to test the changes in this PR.
- [ ] I have updated tests or written new tests as appropriate in this PR.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.

### CODE REVIEW

File: src/views/ModelPlan/HomeTable/index.tsx [hunk 1]
```diff
@@ -356,10 +356,12 @@ const ModelPlansTable = ({
...
-          const crtdlIDs = value
-            .map((crtdl: CRTDLType) => crtdl.idNumber)
-            .join(', ');
-          return crtdlIDs;
+
+          return value.map((crtdl: CRTDLType) => (
+            <ul>
+              <li>{crtdl.idNumber}</li>
+            </ul>
+          ));
```
Comment (garyjzhao) [inline src/views/ModelPlan/HomeTable/index.tsx:364]: Spacing is off.

actual:
image

expect:
image

Comment (garyjzhao) [reply src/views/ModelPlan/HomeTable/index.tsx:364]: oh, while we are at it, the heading is different from figma. `FFS CRs and TDLs` vs figma's `CRs and TDLs`

**Review by garyjzhao — COMMENTED** [review]
Operational solutions tab is overlapping with the sticky nav

image

**Review by garyjzhao — CHANGES_REQUESTED** [review]
see comments above.

another thing, i am not seeing this one being addressed. i am assuming it is because it is nice to have?

image

**Comment by patrickseguraoddball** [comment]
Thanks about to push up changes.  I think Natasha added some of these items after I put up this pr @garyjzhao

File: src/views/HomeNew/components/ModelsBySolutions/card.tsx [hunk 2]
```diff
@@ -38,6 +38,7 @@ const ModelSolutionCard = ({
             <h3 className="model-solution-card__title margin-top-0">
               <UswdsReactLink to={`/models/${id}/read-view`}>
                 {modelName}
+                {abbreviation ? ` (${abbreviation})` : ''}
```
Comment (garyjzhao) [inline src/views/HomeNew/components/ModelsBySolutions/card.tsx:41]: nit: can be abbreviated to `{abbreviation ?? ''}`

Comment (patrickseguraoddball) [reply src/views/HomeNew/components/ModelsBySolutions/card.tsx:41]: We need the space before the abbreviation if it exists, I think that's why I did this

**Review by garyjzhao — APPROVED** [review]
QA feedback have been addressed

**Pull request merged**